multi-arch-builders/coreos-s390x-rhcos-builder.bu

```diff
@@ -0,0 +1,124 @@
+    - name: core
+      ssh_authorized_keys:
+        - ssh-rsa AAAAB3NzaC1yc2EAAAABJQAAAgEAkNiuIximCsbukdOoG99V8dZB3FexJC6CHLuuRUT3xYYXqtcKQXcvb7Nj02zHOiHk34mFpAKNKISFV2pM9qd3TE1Iv7iFkfZanYf4USzHhb5Lm7ZU16H7RyPiCF/YkZ7obmIRb6MOqClGwPTqtYQtOuMR8d35/fJcOijXun1Mt07QG2AHrRVt3Z1gtyxaFz5hln9UUW1Wqby9FKANw3OIZ1xwn/qEZr5dKxGea0mE2PVC5hEerzrC5xIGO00zivKvwtyKnKB5dFCLtvQrPhaYAHKHJBTpMmzjQWgs4MdD6JwFrUSICuiiaVLd8bMGjJgSKsM0t3CJS7J2v9l0ZO0yRd1UdTau6JEbYOjMdE5MRutBUfCLhxkFIfOEQgLf0lE1UxU2XsJQ/zdnMXtExwhSMKfYlOyw1PgP7j8tUuEElY9Pr1pDEm0th4SAwq3+hnFGyP00gddhHY9qYUHi+lwj82lvjPQccSBso+nuJ8L2485e8WUultAUAYQ2MGscPYihkSJXxs0O17k7aLVzz+E8e1UgSuUAcQlrdxson6+P/O9qFnLAKQ+fYJHfoET+RVht4f94lbhr15M9IJGHZGQexwqshuABhsE2DKtT1/nizsl+85CpF1Esj3w1mEGhuDEiuCmC7wI0i12+YMuuYMdGcM75mhWFD0QjIjwcQjmmZyk= jan.schintag@de.ibm.com
+kernel_arguments:
+  should_exist:
+    # enables Secure Execution Host
+    - prot_virt=1
+storage:
+  directories:
+    - path: /home/builder/.config/systemd/user/default.target.wants
+      user:
+        name: builder
+      group:
+        name: builder
+  files:
+    - path: /etc/hostname
+      mode: 0644
+      overwrite: true
+      contents:
+        inline: rdu-z15-l21
+    - path: /home/core/create-secex-data.sh
+      mode: 0744
+      user:
+        name: core
+      group:
+        name: core
+      contents:
+        inline: |
+            #!/bin/bash
+
+            set -e
+
+            DISK_PART="/dev/disk/by-partuuid/80442b5f-01"
+            DISK_FCP="0.0.a800"
+            DISK_WWWN="0x5005076810154e60"
+            DISK_LUN="0x0000000000000000"
+            MNTP="/mnt/secex-data"
+            TARBALL="secex.tar"
+
+            if ! $(whoami | grep -q "root"); then
+                echo "This script must be run as root"
+                exit 1
+            fi
+
+            if ! $(lszdev | grep -q "${DISK_LUN}"); then
+                echo "Adding LUN to system"
+                echo "${DISK_LUN}" > /sys/bus/ccw/drivers/zfcp/${DISK_FCP}/${DISK_WWWN}/unit_add
+            fi
+
+            if ! $(mountpoint -q "${MNTP}"); then
+                echo "Mounting disk"
+                mkdir -p "${MNTP}"
+                mount "${DISK_PART}" "${MNTP}"
+            fi
+
+            echo "Copying tarball from disk to builder"
+            cp "${MNTP}/${TARBALL}" "/var/home/builder/${TARBALL}"
+            chown builder:builder "/var/home/builder/${TARBALL}"
+
+            if ! $(sudo -u builder -H /bin/bash -c "cd /var/home/builder; podman volume list" | grep -q secex-data); then
+                echo "Creating volume secex-data"
+                sudo -u builder -H /bin/bash -c "cd /var/home/builder; podman volume create --label=persistent secex-data"
+            fi
+
+            echo "Importing tarball into volume"
+            sudo -u builder -H /bin/bash -c "cd /var/home/builder; podman volume import secex-data /var/home/builder/${TARBALL}"
+    - path: /etc/systemd/secex-data-volume.service
```

Comment:
since this is a system systemd unit (not a user unit) let's just use a [systemd directive](https://coreos.github.io/butane/examples/#systemd-units) rather than create the file manually. It has the benefit of being able to just set `enabled: true` without having to manually create the symlink below.

Reply:
I updated the PR accordingly